.github/workflows/ci.yml

```diff
@@ -0,0 +1,122 @@
+name: CI
+
+on:
+  pull_request:
+    branches: ["*"]
+  push:
+    branches: ["master"]
+    tags: ["v*"]
+
+# env:
```

Comment:
TODO: remove (we don't need as much memory it seems given that we're running jobs in parallel)